webpack.config.js

```diff
@@ -91,6 +91,13 @@ Encore
         }
     )
 
+    .copyFiles(
```

Comment:
This is not needed, and should be undone.

Reply:
undone.